### ISSUE TITLE
igx-nav-drawer item background on hover changed

### ISSUE THREAD

**Issue by kdinev** [IgniteUI/igniteui-angular#10478]
## Description  
The `igx-navdrawer` has a different background color on hover in 13.0 from 12.2.

 * igniteui-angular version: 13.0.0-alpha.7
 * browser: all

## Steps to reproduce  

1. Create an angular view with an `igx-navdrawer`.
```html
  <igx-nav-drawer [pinThreshold]="1600" id="navigation" #drawer>
    <ng-template igxDrawer>
      <nav>
        <span igxDrawerItem [isHeader]="true">Bellum Gens</span>
        <span igxDrawerItem igxRipple *ngIf="authUser" routerLinkActive="igx-nav-drawer__item--active" routerLink="team/myteams">
          <igx-icon>people</igx-icon>
          <span i18n>My Teams</span>
          <igx-badge class="notifications-badge" *ngIf="teams?.length > 0" [value]="teams.length" type="info"></igx-badge>
        </span>
        <span igxDrawerItem igxRipple *ngIf="authUser" routerLinkActive="igx-nav-drawer__item--active" routerLink="notifications">
          <igx-icon>notifications</igx-icon>
          <span i18n>Notifications</span>
          <igx-badge class="notifications-badge" *ngIf="unreadNotifications > 0" [value]="unreadNotifications" type="info"></igx-badge>
        </span>
        <span igxDrawerItem igxRipple *ngIf="authUser" routerLinkActive="igx-nav-drawer__item--active" routerLink="strategies/user">
          <igx-icon>person</igx-icon>
          <span i18n>My Strategies</span>
        </span>
        <igx-divider *ngIf="authUser"></igx-divider>
        <span igxDrawerItem igxRipple routerLinkActive="igx-nav-drawer__item--active" routerLink="strategies/public">
          <igx-icon>playlist_add_check</igx-icon>
          <span i18n>CS:GO Strategies</span>
        </span>
        <span igxDrawerItem igxRipple routerLink="/" fragment="partners">
          <igx-icon>people_alt</igx-icon>
          <span i18n>Partners</span>
        </span>
        <span igxDrawerItem igxRipple routerLink="/" fragment="aboutus">
          <igx-icon>chat</igx-icon>
          <span i18n>Contacts</span>
        </span>
      </nav>
    </ng-template>

    <ng-template igxDrawerMini *ngIf="drawer.pin">
      <nav>
        <span igxDrawerItem [isHeader]="true">&nbsp;</span>
        <span igxDrawerItem igxRipple *ngIf="authUser" routerLinkActive="igx-nav-drawer__item--active" routerLink="team/myteams" i18n-title title="My Teams">
          <igx-icon>people</igx-icon>
          <igx-badge class="notifications-badge" *ngIf="teams?.length > 0" [value]="teams.length" type="info"></igx-badge>
        </span>
        <span igxDrawerItem igxRipple *ngIf="authUser" routerLinkActive="igx-nav-drawer__item--active" routerLink="notifications" i18n-title title="Notifications">
          <igx-icon>notifications</igx-icon>
          <igx-badge class="notifications-badge" *ngIf="unreadNotifications > 0" [value]="unreadNotifications" type="info"></igx-badge>
        </span>
        <span igxDrawerItem igxRipple *ngIf="authUser" routerLinkActive="igx-nav-drawer__item--active" routerLink="strategies/user" i18n-title title="My Strategies">
          <igx-icon>person</igx-icon>
        </span>
        <igx-divider *ngIf="authUser"></igx-divider>
        <span igxDrawerItem igxRipple routerLinkActive="igx-nav-drawer__item--active" routerLink="strategies/public" i18n-title title="CS:GO Strategies">
          <igx-icon>playlist_add_check</igx-icon>
        </span>
        <span igxDrawerItem igxRipple routerLink="/" fragment="partners" i18n-title title="Partners">
          <igx-icon>people_alt</igx-icon>
        </span>
        <span igxDrawerItem igxRipple routerLink="/" fragment="aboutus" i18n-title title="About us">
          <igx-icon>chat</igx-icon>
        </span>
      </nav>
    </ng-template>
  </igx-nav-drawer>
```

2. Define the following theme:
```scss
$primary: white !default;
$secondary: #FF0A3A !default;
$background: #192053;
$banner-background: #2C3D93;
$background-gradient: linear-gradient(to right, $background, #04081D);
$background-gradient-lighter: linear-gradient(135deg, #2f3978 0%, #10193f 100%);
$progress-success: #13FF21;

$app-palette: igx-palette($primary, $secondary);

@include igx-dark-theme(
  $palette: $app-palette
);

/* Nav Drawer */
$navdrawer-theme: igx-navdrawer-theme(igx-palette($background, $secondary),
                                      $background: $background-gradient,
                                      $item-text-color: $primary,
                                      $item-icon-color: $primary,
                                      $item-active-text-color: $secondary,
                                      $item-active-icon-color: $secondary,
                                      $item-hover-text-color: $secondary,
                                      $item-hover-icon-color: $secondary);
@include igx-css-vars($navdrawer-theme);
/* END Nav Drawer */
```

## Result  
![image](https://user-images.githubusercontent.com/1472513/141304028-b9f235ad-c430-4513-baf3-27b3eee8134a.png)

## Expected result  
![image](https://user-images.githubusercontent.com/1472513/141304073-0e90dbbd-c2e7-403a-99e9-47112a2aafee.png)

## Attachments  
Attach a sample if available, and screenshots, if applicable.

**Comment by kdinev** [comment]
Fixed this by adding `$item-hover-background: rgba(0, 0, 0, 0.11)` to my nav-drawer theme.

However, I first tried adding `$schema: $dark-schema` and that didn't quite do the trick. It added some transparency, but it wasn't a lot and the bg-color was white.

**Comment by simeonoff** [comment]
This is strange as I can't reproduce it with the correct schema and palette combo.

**Comment by kdinev** [comment]
The transparency is definitely fixed with `13.0.0` when `$dark-schema` is provided. Closing:

![image](https://user-images.githubusercontent.com/1472513/143880416-91c6ba73-0af7-4895-9e6f-20415f4bdeb7.png)
